lumicks/pylake/correlated_stack.py

```diff
@@ -241,54 +284,46 @@ def get_image(self, channel="rgb"):
 
         return np.stack([frame.data[slc] for frame in self], axis=0).squeeze()
 
+    @_deprecate_plot_arguments
     def plot(
         self,
-        frame=0,
         channel="rgb",
-        show_title=True,
         axes=None,
-        adjustment=ColorAdjustment.nothing(),
         *,
+        frame=0,
         image_handle=None,
+        adjustment=ColorAdjustment.nothing(),
+        show_title=True,
         **kwargs,
     ):
-        """Plot image from image stack
+        """Plot a frame from the image stack for the requested color channel(s)
 
         Parameters
         ----------
+        channel : {"red", "green", "blue", "rgb"}, optional
+            Color channel to plot.
+        axes : mpl.axes.Axes, optional
+            If supplied, the axes instance in which to plot.
         frame : int, optional
             Index of the frame to plot.
-        channel : 'rgb', 'red', 'green', 'blue', None; optional
-            Channel to plot for RGB images (None defaults to 'rgb')
-            Not used for grayscale images
-        show_title : bool, optional
-            Controls display of auto-generated plot title
-        axes : mpl.axes.Axes or None
-            If supplied, the axes instance in which to plot.
         image_handle : matplotlib.image.AxesImage or None
             Optional image handle which is used to update plots with new data rather than
             reconstruct them (better for performance).
+        adjustment : lk.ColorAdjustment
+            Color adjustments to apply to the output image.
+        show_title : bool, optional
+            Controls display of auto-generated plot title
         **kwargs
-            Forwarded to :func:`matplotlib.pyplot.imshow`.
+            Forwarded to :func:`matplotlib.pyplot.imshow`. Only used, if `image_handle` is not
+            provided.
```

Comment:
Generally I prefer to avoid speaking in the negative sense for stuff like this, I think it's clearer

```suggestion
            Forwarded to :func:`matplotlib.pyplot.imshow`. These arguments are ignored
            if `image_handle` is provided.
```

Reply:
👍